### PR TITLE
Add Design Studio and description needed labels to README

### LINKED ISSUE #30: Add some new labels
A `needs_description` label to workflow -> needs action
For cases where the issue is not properly described, we put this label and assign someone (normally the person who created the issue) to write it.

A `Design Studio` label to Domain labels

Proposed [here](https://github.com/resmio/git-along/pull/28#issuecomment-250387008)

### LINKED ISSUE #34: Add design studio info to the repo
Just copy the description @fabiennepimenta provided [here](https://github.com/resmio/git-along/issues/23#issuecomment-250720160) to some place in the repo that makes sense.

### PR DESCRIPTION
**Pull request by JanNash**
fixes #30 
maybe fixes #34 ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +63,7 @@ Workflow labels define the position of an issue or PR in out process:
 
 **Needs action labels**  
 Color: gold (#FFD700)  
+- `description needed`: This issue is not described well enough. Typically, it would be reassigned to the person who created it, so they can describe it better. This is an important step to make the issues the **_single source of truth_**.
```
Comment (JannKleen) [inline README.md]: Isn't that just a sub-label of 'requirements needed' ?

Comment (JanNash) [inline README.md]: I was thinking about that myself, not sure. @menor ?

Comment (menor) [inline README.md]: The idea was to use before we even think about requirements, to avoid having issues with no description that just sit there.

Comment (JannKleen) [inline README.md]: 👍 let's try it out

**Comment by JanNash** [comment]
@fabiennepimenta are you onboard with that, too?

**Comment by JanNash** [comment]
lgtm, I guess :)

**Pull request merged**